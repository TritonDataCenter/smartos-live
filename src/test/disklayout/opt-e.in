SCSI	disk0	ATA	INTEL SSDSC2BX01	1600321314816	no	yes
SCSI	disk1	ATA	INTEL SSDSC2BX01	1600321314816	no	yes
SCSI	disk2	ATA	INTEL SSDSC2BX01	1600321314816	no	yes
SCSI	disk3	ATA	INTEL SSDSC2BX01	1600321314816	no	yes
SCSI	disk4	ATA	INTEL SSDSC2BX01	1600321314816	no	yes
SCSI	disk5	ATA	INTEL SSDSC2BX01	1600321314816	no	yes
SCSI	disk6	ATA	INTEL SSDSC2BX01	1600321314816	no	yes
SCSI	disk7	ATA	INTEL SSDSC2BX01	1600321314816	no	yes
USB	key0	USBMFG	USB Key	15737028608	yes	no
