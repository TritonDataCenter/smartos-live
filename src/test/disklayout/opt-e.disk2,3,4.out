{
	"spares": [
		{
			"name": "disk7",
			"vid": "ATA",
			"pid": "INTEL SSDSC2BX01",
			"size": "1600321314816",
			"solid_state": true
		}
	],
	"vdevs": [
		{
			"type": "mirror",
			"devices": [
				{
					"name": "disk0",
					"vid": "ATA",
					"pid": "INTEL SSDSC2BX01",
					"size": "1600321314816",
					"solid_state": true
				},
				{
					"name": "disk1",
					"vid": "ATA",
					"pid": "INTEL SSDSC2BX01",
					"size": "1600321314816",
					"solid_state": true
				}
			]
		},
		{
			"type": "mirror",
			"devices": [
				{
					"name": "disk5",
					"vid": "ATA",
					"pid": "INTEL SSDSC2BX01",
					"size": "1600321314816",
					"solid_state": true
				},
				{
					"name": "disk6",
					"vid": "ATA",
					"pid": "INTEL SSDSC2BX01",
					"size": "1600321314816",
					"solid_state": true
				}
			]
		}
	],
	"capacity": 3200642629632
}
